shell/browser/api/electron_api_app.cc

```diff
@@ -617,6 +616,22 @@ void App::OnPreMainMessageLoopRun() {
   }
 }
 
+void App::OnPreCreateThreads() {
+  DCHECK(!content::GpuDataManager::Initialized());
+  content::GpuDataManager* manager = content::GpuDataManager::GetInstance();
+  manager->AddObserver(this);
+  gpu_data_manager_initialized_ = true;
```

Comment:
I'm surprised we need both this and `content::GpuDataManager::Initialized()`?

Reply:
Oops thats bad 😆